meta/main.yml

```diff
@@ -0,0 +1,24 @@
+galaxy_info:
+  author: Janne Heß
+  description: Set up an Icingaweb2 installation
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: CC-BY-SA
+
+  min_ansible_version: 2.4
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - xenial
+        - bionic
+
+  galaxy_tags:
+    - icinga
+    - icinga2
+    - icingaweb
+    - icingaweb2
+    - monitoring
+    - webui
+
+dependencies: []
```

Comment:
php verlinken

Reply:
Das eskaliert doch immer, oder?

Reply:
ja weil wir unsere Rollen nicht aus galaxy beziehen.
Könnten wir theoretisch mal, aber die sind da auch irgendwie nicht aktuell, scheinen nicht in github zu hooken.